pkgs/development/python-modules/polling/default.nix

```diff
@@ -1,40 +1,38 @@
 {
   lib,
   buildPythonPackage,
-  fetchFromGitHub,
+  pytest,
+  fetchPypi,
   setuptools,
   wheel,
   mock,
-  pytestCheckHook,
 }:
 
 buildPythonPackage rec {
   pname = "polling";
   version = "0.3.0";
   pyproject = true;
 
-  src = fetchFromGitHub {
-    owner = "justiniso";
-    repo = "polling";
-    rev = "v${version}";
-    hash = "sha256-Qy2QxCWzAjZMJ6yxZiDT/80I2+rLimoG8/SYxq960Tk=";
+  src = fetchPypi {
+    inherit version;
+    pname = "polling";
+    hash = "sha256-wPH4BIQ/544J9vxt5vuLEfc74KzOnN4mKxjkp61ABaY=";
   };
 
   nativeBuildInputs = [
     setuptools
     wheel
   ];
 
-  pythonImportsCheck = [ "polling" ];
-
-  nativeCheckInputs = [
+  propagatedBuildInputs = [
     mock
-    pytestCheckHook
+    pytest
```

Comment:
Why do we propagate pytest?

Reply:
[It looks like I should just disable pytestCheckPhase](https://github.com/NixOS/nixpkgs/pull/470777#discussion_r2623172128). That being said, since the source is missing, it might make sense to drop the package. But, I think there's one dependency by linode-api.